src/amo/actions/reviews.js

```diff
@@ -132,6 +135,50 @@ export function fetchReviews({
   };
 }
 
+type FetchRatingSummaryParams = {|
+  addonId: number,
+  errorHandlerId: string,
+|};
+
+export type FetchRatingSummaryAction = {|
+  type: typeof FETCH_RATING_SUMMARY,
+  payload: FetchRatingSummaryParams,
+|};
+
+export function fetchRatingSummary({
+  addonId,
```

Comment:
I see in the api docs, that this could this be addon slug. I am wondering if this could be an alternate arg to pass in or does that make it too messy (I do see you use this addonId as a key in some places)? Just wondering : )